### PR TITLE
makefile goal to display the latest e2e project names

### PR DESCRIPTION
**Pull request by xcoulon**
The goal of `display-eval` is to provide a convenient way to retrieve the latest host and member operator namespaces on a cluster, and optionally, to export them as env vars:
```
$ make display-eval
export HOST_NS=host-operator-1574156115
export MEMBER_NS=member-operator-1574156115
export REGISTRATION_SERVICE_NS=$HOST_NS
# Run this command to configure your shell:
# eval $(make display-eval)
```

The `host-operator-xxx` and `member-operator-xxx` projects are now labelled with `app=host-operator` and `app=member-operator` respectively the latest occurrence is retained. 
By querying the cluster, we can use this goal alone, without interfering with the `HOST_NS` and `MEMBER_NS` variables set in the `test.mk` file.

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -304,3 +307,11 @@ else
 		sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g' ${E2E_REPO_PATH}/deploy/operator.yaml | oc apply -f -
     endif
 endif
+
+.PHONY: display-eval
+display-eval:
+	@echo 'export HOST_NS=$(shell oc get projects -l app=host-operator --output=name --no-headers=true -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)'
```
Comment (MatousJobanek) [inline make/test.mk]: I think that you could make it shorter and drop the jsonpath as well as the --no-headers param:
```suggestion
	@echo 'export HOST_NS=$(shell oc get projects -l app=host-operator --output=name  | cut -f2 -d"/" | sort | tail -n 1)'
```
but it's only my proposal - you don't have to use it as your solution is fine too ;-)

File: make/test.mk [hunk 2]
```diff
@@ -304,3 +307,11 @@ else
...
+	@echo 'export MEMBER_NS=$(shell oc get projects -l app=member-operator --output=name --no-headers=true -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)'
```
Comment (MatousJobanek) [inline make/test.mk]:
```suggestion
	@echo 'export MEMBER_NS=$(shell oc get projects -l app=member-operator --output=name | cut -f2 -d"/" | sort | tail -n 1)'
```

File: make/test.mk (hunk 1 above)
Comment (xcoulon) [reply make/test.mk]: well, I kinda like the jsonpath expression, so I'll keep it as-is for now if it's ok for you ;)

File: make/test.mk (hunk 2 above)
Comment (xcoulon) [reply make/test.mk]: same here, keeping the `jsonpath` for now, but indeed, the `--no-headers=true` is not needed, so I'll remove it.

Comment (xcoulon) [reply make/test.mk]: done in 689670f

File: make/test.mk (hunk 1 above)
Comment (xcoulon) [reply make/test.mk]: removed the `--no-headers=true` arg in 689670f

**Comment by xcoulon** [comment]
> WOW, this is very nice improvement 👍 🥇

thanks @MatousJobanek!

**Pull request merged**